nautobot/core/templates/base.html

```diff
@@ -85,22 +88,25 @@ <h4><i class="mdi mdi-alert"></i> Maintenance Mode</h4>
                 </div>
                 <div class="col-xs-4 text-right noprint">
                     <p class="text-muted">
+                        <a href="#theme_modal" data-toggle="modal" data-target="#theme_modal" id="btn-theme-modal"><span class="mdi mdi-theme-light-dark text-primary">Theme</span></a> &middot;
                         <i class="mdi mdi-book-open-page-variant text-primary"></i>
                         {% if settings.BRANDING_URLS.docs %}
-                             <a href="{{ settings.BRANDING_URLS.docs }}">Docs</a>
+                            <a href="{{ settings.BRANDING_URLS.docs }}">Docs</a>
                         {% else %}
                             <a href="{% static 'docs/index.html' %}">Docs</a>
                         {% endif %}
                         &middot;
-                        <i class="mdi mdi-cloud-braces text-primary"></i> <a href="{% url 'api_docs' %}">API</a> &middot;
-                        <i class="mdi mdi-graphql text-primary"></i> <a href="{% url 'graphql' %}">GraphQL</a> &middot;
-                        <i class="mdi mdi-xml text-primary"></i> <a href="{{ settings.BRANDING_URLS.code }}">Code</a> &middot;
-                        <i class="mdi mdi-lifebuoy text-primary"></i> <a href="{{ settings.BRANDING_URLS.help }}">Help</a>
+                        <span class="mdi mdi-cloud-braces text-primary"></span> <a href="{% url 'api_docs' %}">API</a> &middot;
+                        <span class="mdi mdi-graphql text-primary"></span> <a href="{% url 'graphql' %}">GraphQL</a> &middot;
+                        <span class="mdi mdi-xml text-primary"></span> <a href="{{ settings.BRANDING_URLS.code }}">Code</a> &middot;
+                        <span class="mdi mdi-lifebuoy text-primary"></span> <a href="{{ settings.BRANDING_URLS.help }}">Help</a>
                     </p>
                 </div>
             </div>
         </div>
     </footer>
+{% include 'modals/modal_theme.html' with name='theme'%}
+<script src="{% static 'js/theme.js' %}"></script>
```

Comment:
Do we need similar additions to `nautobot/core/templates/admin/base.html` so that the admin pages also support dark mode?

Reply:
Yes we do.  I'll add it.
For future considerations, there's a few different "base.html" type files I've had to go in and modify.  I think ideally in the future, we consolidate these into one "true" base file.

Reply:
Actually in looking at this now, it already appears the footers are out of sync.  I'll update this as well.